_site/assets/img/menu-down.svg

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright © 2015 Apple Inc. All rights reserved. -->
```

Comment:
Depending on the licences we should have a dedicated document with all the licences we are using.

Reply:
Good point, i reckon i could replace that chevron with a creative commons one

Reply:
Maybe this could be helpful: https://feathericons.com/